src/test/java/utils/VacancySQLDataProvider.java

```diff
@@ -0,0 +1,41 @@
+package utils;
+
+import data.EmployeeSQLData;
+import data.JobVacancySQLData;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class VacancySQLDataProvider extends SQLDataProvider{
+    public static List<JobVacancySQLData> vacancyPool = new ArrayList<>();
+
+    public static void getVacancyPool() {
+        String url = "jdbc:mysql://test.biz.ua:3306/orangehrm_mysql";
+
+        String vcQuery = "SELECT ohrm_job_vacancy.hiring_manager_id, ohrm_job_vacancy.name " +
+                "FROM ohrm_job_vacancy;";
+
+        try (Connection con = DriverManager.getConnection(URL, DBNAME, DBPASSWORD);
+             Statement stmt = con.createStatement();
+             ResultSet rs = stmt.executeQuery(vcQuery)) {
+            while (rs.next()) {
+                JobVacancySQLData vacancy = new JobVacancySQLData();
+                vacancy.setHiring_manager_id(rs.getString("hiring_manager_id"));
+                vacancy.setName(rs.getString("name"));
+                vacancyPool.add(vacancy);
+            }
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static EmployeeSQLData getEmployeesByJobVacancyName (String jobVacancyName){
```

Comment:
EmployeeSQLData from VacancyProvider - the same

Reply:
moved to EmpProvider